apps/admin-gui/src/app/vos/pages/group-detail-page/group-overview/group-overview.component.ts

```diff
@@ -96,10 +96,9 @@ export class GroupOverviewComponent implements OnInit {
     this.navItems = [];
 
     if (
-      this.guiAuthResolver.isAuthorized(
-        'getCompleteRichMembers_Group_List<String>_boolean_policy',
-        [this.group]
-      )
+      this.guiAuthResolver.isAuthorized('getMembersPage_Vo_MembersPageQuery_List<String>_policy', [
```

Comment:
I'm not sure, but isn't the correct policy this one? Previous policy checks VO Admin/Observer and Group Admin/Observer in VO but this one (with group prefix) checks the Group Admin/Observer role for a given group.
```suggestion
      this.guiAuthResolver.isAuthorized('group-getMembersPage_Vo_MembersPageQuery_List<String>_policy', [
```